tests/posix/common/src/pthread.c

```diff
@@ -587,12 +587,15 @@ ZTEST(posix_apis, test_pthread_descriptor_leak)
 	pthread_t pthread1;
 	pthread_attr_t attr;
 
+	zassert_ok(pthread_attr_init(&attr));
```

Comment:
This looks correct according to the POSIX spec. I wonder if you'd rather call `pthread_attr_destroy` in the loop? That seems like an equally valid option.

Reply:
Either way is fine; thanks for fixing the old code with `pthread_attr_init` in the loop.